service/src/test/java/com/commercetools/pspadapter/payone/notification/common/CaptureNotificationProcessorTest.java

```diff
@@ -66,12 +63,21 @@ public void setUp() throws Exception {
     @Test
     @SuppressWarnings("unchecked")
     public void processingPendingNotificationAboutUnknownTransactionAddsChargeTransactionWithStatePending() throws Exception {
+        // note, this case tests un-documented (actually, documented, but not clearly) feature:
+        // it is not clear, could it ever be that txaction=capture has transaction_state=pending.
+        // Doc says [PAYONE Platform Channel Server API, Edition: 2017-08-29, Version: 2.92, Chapter 4.2.2 List of status (transaction_status)]:
+        //   > The parameter “transaction_status” is currently introduced with event-txaction “appointed” only.
+        // but later:
+        //   > Other event-txaction with parameter “transaction_status” may follow (e.g. “paid”, “debit”, ...).
+        //   > The new “txaction” can then be “paid/pending”, “paid/completed”, ... or “failed/completed”.
+        //
+        // So, even if Payone changes API and such case happens - we expect to switch CT transaction state to PENDING, not SUCCESS.
```

Comment:
I am really confused by the comments above.
What does it mean "we expect to switch CT transaction state to PENDING, not SUCCESS."?
Where is it shows SUCCESS in the line of code below?
What does it mean "we expect"? So it's expected to be changed at one point? when? why not changed now?

Reply:
> What does it mean "we expect to switch CT transaction state to PENDING, not SUCCESS."?
> What does it mean "we expect"? So it's expected to be changed at one point? when? why not changed now?

ok, we don't expect, we actually set it to this value. 3de9fe90

> Where is it shows SUCCESS in the line of code below?

the comment says opposite: _switch CT transaction state to **PENDING**_, which is asserted in the function below (_...ChargeTransactionWithState**Pending**_) (i agree the function name is too long,  though it could be re-factored as a separate task/pr)